frontend/packages/topology/src/data-transforms/DataModelExtension.tsx

```diff
@@ -1,29 +1,95 @@
-import * as React from 'react';
+import { useContext, useRef, useEffect, useMemo } from 'react';
+import {
+  WatchK8sResources,
+  WatchK8sResourcesGeneric,
+  WatchK8sResource,
+  TopologyDataModelFactory as DynamicTopologyDataModelFactory,
```

Comment:
we no longer need to differentiate the dynamic extension

```suggestion
  TopologyDataModelFactory,
```